pkg/controllers/dynakube/istio/reconciler.go

```diff
@@ -157,40 +221,62 @@ func (r *reconciler) reconcileIPServiceEntry(ctx context.Context, ipHosts []dtcl
 	return nil
 }
 
+func (r *reconciler) cleanupIPServiceEntry(ctx context.Context, component string) error {
+	entryName := BuildNameForIPServiceEntry(r.client.Owner.GetName(), component)
+
+	err := r.client.DeleteServiceEntry(ctx, entryName)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
 func (r *reconciler) reconcileFQDNServiceEntry(ctx context.Context, fqdnHosts []dtclient.CommunicationHost, component string) error {
 	owner := r.client.Owner
 	entryName := BuildNameForFQDNServiceEntry(owner.GetName(), component)
 
 	if len(fqdnHosts) != 0 {
-		meta := buildObjectMeta(
+		objectMeta := buildObjectMeta(
```

Comment:
out of curiosity, why do we need this change here? it seems unrelated to this PR.

Reply:
true, but I got the hint `Variable 'meta' collides with imported package name` so I thought it might improve readability...

Reply:
okey, it should work.